.gitignore

```diff
@@ -33,3 +33,6 @@ supervisord.pid
 tags
 /config
 /data
+*.skey
+*.vkey
+*.cert
```

Comment:
Is it useful to add this to `.gitignore`? If not:

```suggestion
```

Reply:
I almost accidentally committed some generated keys, figure it's probably not a good idea to put keys in repo unless you force it. Thoughts?

Reply:
Sure, that's fine.